doc/release-notes/0.8/0.8.md

```diff
@@ -0,0 +1,76 @@
+<!--
+* Copyright (c) 2017, 2018 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Eclipse OpenJ9 version 0.8 release notes
+
+## Supported platforms
+
+OpenJDK binaries that contain the Eclipse OpenJ9 virtual machine are available from the
+[AdoptOpenJDK community](https://adoptopenjdk.net/?variant=openjdk8-openj9). These builds are
+being tested against the OpenJDK test suite.
+
+The following platforms are available:
+
+|Operating system | Platform architecture | Comments                                    |
+|-----------------|-----------------------|---------------------------------------------|
+|Linux            | x86, IBM Power, IBM Z | Windows subsytem for Linux is not supported |
+|AIX              | IBM Power             | Untested                                    |
+|Windows          | x86                   | Untested                                    |
```

Comment:
We can have a Windows test build at AdoptOpenJDK, there is a machine configuration problem however, currently only a single run of the OpenJDK regression tests can run cleanly before a reboot is needed, 2nd run fails with some permission issues.  So, we can report on open test results for the Windows platform if needed.

Reply:
for Java 8? I don't see any Adopt Windows builds for Java 8 yet. This release is only supporting Java 8.

Reply:
Speaking of which, the release notes don't mention this at all.

Reply:
Because of the machine issue, the windows test jobs were moved to the "work in progress" view:

However, I see we only have the hotspot run: https://ci.adoptopenjdk.net/view/work%20in%20progress/job/openjdk_test_x86-64_windows/250/  

as there is no successful openj9 builds at AdoptOpenJDK at the moment to trigger the tests for the openj9 test build:
https://ci.adoptopenjdk.net/view/work%20in%20progress/job/openjdk8_j9_openjdktest_x86-64_windows/

Reply:
Other links: in case you wish to reference in notes (though likely that is too much detail to include):
SV: https://ci.adoptopenjdk.net/view/System%20tests/
OpenJDK regression: https://ci.adoptopenjdk.net/view/OpenJDK%20tests/job/openjdk8_j9_openjdktest_x86-64_linux/

Reply:
Changes:

- Updated the first paragraph to point to the OpenJ9 release plan at Eclipse.
- Added a sentence to supported platforms to make it clearer that we are talking V8, not V9:
*OpenJDK Version 8 binaries containing the Eclipse OpenJ9 virtual machine are supported with this release.*